src/main/resources/anniv6Resources/localization/eng/Junkyard/UIstrings.json

```
@@ -0,0 +1,14 @@
+{
+  "${ModID}:Junkyard": {
+    "TEXT": [
+      "The Junkyard",
+      "Lots of things to pick up! Gain 2 card rewards per combat but cannot skip card rewards. Also contains unique encounters.",
```

```suggestion
      "Lots of things to pick up! Gain 2 card rewards per combat but you can't skip card rewards. Also contains unique encounters.",
```
You can leave the unique encounters line if you want to, but it's not really needed. Could also be flavorful about it `Beware of the trashy denizens!`